mysql_servers.json

```diff
@@ -185,6 +185,41 @@
       "max_restarts": "1",
       "min_uptime": "2m"
     },
+    {
```

Comment:
I think that, as a general rule, we usually prefer to append new services to the end of this JSON file, so that the `pm2` ordinal shortcuts for everything else stay the same and people's muscle memory isn't interrupted. But this is probably close enough to the end not to matter, I doubt many people have internalised the numbers for the items that follow these.

Also, in this case it leads to the ordinals for other things being different depending on whether `servers.json` or `mysql_servers.json` is used. I think we try to avoid that too.

Reply:
Ah, makes sense. Should I add this to `servers.json` as well? I had totally forgotten about that one, do people regularly use that over `mysql_servers.json`?

Reply:
> Should I add this to `servers.json` as well? I had totally forgotten about that one, do people regularly use that over `mysql_servers.json`?

I still use `servers.json` a lot of the time, but only from habit formed before we had the MySQL one. Maybe we should get rid of it, but as long as it still exists it's worth keeping them both updated I guess?